GliaWidgets/Lib/Upload/FileUploader.swift

```diff
@@ -95,8 +95,15 @@ class FileUploader {
         updateLimitReached()
     }
 
-    func removeAllUploads() {
-        uploads.removeAll()
+    func removeSucceededUploads() {
+        uploads.removeAll(where: { upload in
+            switch upload.state.value {
+            case .uploaded:
+                return true
+            default:
+                return false
+            }
+        })
```

Comment:
Probably we could simplify this closure to 
```suggestion
        uploads.removeAll(where: { $0..state.value == .uploaded })
```

Reply:
Actually no, this wouldn't compile. 
I remember trying to make it a one-liner but I was unable to do so. Tried so again after coming back from vacation but still nothing. I don't think it's worth my time anymore or is critical in any regard, so I'll leave it as is.

Reply:
It can't be done because FileUpload.State has parameters for enum cases.

Reply:
Yes, exactly